Sources/VSM/StateContainer/StateObserving.swift

```diff
@@ -101,7 +101,7 @@ public extension StateObserving {
     ///   - nextState: The action to be debounced before invoking
     ///   - dueTime: The amount of time required to pass before invoking the most recent action
     func observe(
-        _ nextState: @escaping () -> State,
+        _ nextState: @escaping @autoclosure () -> State,
```

Comment:
Unrelated to main PR purpose, but was an omission from the VSM 1.0 refactoring. This is a "spelling-only" breaking change.